test/extended/pods/graceful-shutdown.go

```diff
@@ -340,7 +340,7 @@ func createTestBed(ctx context.Context, oc *exutil.CLI) {
 	Expect(err).NotTo(HaveOccurred())
 	err = exutil.WaitForServiceAccountWithSecret(
 		oc.AdminKubeClient().CoreV1().ServiceAccounts(namespace),
-		serviceAccountName)
+		serviceAccountName, true)
```

Comment:
I'm not clear on why you can hard-code `true` here and `false` in the earlier CPU-partitioning file.  Can you add some notes to the commit message explaining those choices?

Reply:
Sure, I'm not sure which SA here, but I just want to keep the same logic, `true` here means `false` in there: https://github.com/openshift/origin/pull/28482/files#diff-35a89a7a7362642eebb559fb8564e857b00d6f7dd6322c3adabaf1adbd609d35L1086
```go
		if hasDockercfg || !checkSecret {
			return true, nil
		}
```

Reply:
Update `WaitForServiceAccountWithSecret` function instead.